src/components/breadcrumb/_breadcrumb.scss

```diff
@@ -74,19 +74,10 @@
 @mixin breadcrumb--x {
   .#{$prefix}--breadcrumb {
     @include type-style('body-short-01');
-
     display: inline;
-
-    @if feature-flag-enabled('grid') {
-      @include carbon--breakpoint(md) {
-        display: flex;
-        flex-wrap: wrap;
-      }
-    } @else {
-      @include breakpoint(bp--xs--major) {
-        display: flex;
-        flex-wrap: wrap;
-      }
+    @include carbon--breakpoint(md) {
+      display: flex;
```

Comment:
Should we keep the `grid` check? I don't know if some people are using new grid with components-x on so might be worthwhile to keep it around until the swap 👍

Reply:
OK brought it back - BTW we are not keeping old grid in `v10`, right?